modules/scheduler/batch-job-template/templates/batch-job-base.yaml.tftpl

```diff
@@ -5,8 +5,10 @@ taskGroups:
         - barrier:
             name: "wait-for-node-startup"
         %{~ endif ~}
+        %{~ for runnable in runnables ~}
         - script:
-            text: ${indent(12, chomp(yamlencode(runnable)))}
+            text: ${indent(20, chomp(yamlencode(runnable)))}
```

Comment:
Interesting that indent changed from 12 to 20. Why is that?

Reply:
I don't remember what exactly happened but at one point during work on this I got it into my head that I needed the extra indentation to make the formatting come out properly. I probably had the wrong tildes ~ around some of my ifs or fors. I changed it back to 12 now and toolkit still generates well-formed job specs.